port/zephyr/Kconfig

```diff
@@ -6,12 +6,110 @@ menuconfig GOLIOTH_FIRMWARE_SDK
 	select POSIX_API # libcoap
 	select REBOOT if BOOTLOADER_MCUBOOT
 	select REQUIRES_FULL_LIBC # libcoap
+	select MBEDTLS
+	select MBEDTLS_DTLS if MBEDTLS_BUILTIN
+	select NET_SOCKETS
+	select NET_UDP
+	select ZCBOR
+	imply DNS_RESOLVER if NET_NATIVE
+	imply NET_SOCKETS_SOCKOPT_TLS
+	imply NET_SOCKETS_ENABLE_DTLS
+	help
+	  Enable library for communication with Golioth cloud.
 
 if GOLIOTH_FIRMWARE_SDK
 
 menu "Golioth SDK Configuration"
 
-rsource '../../src/Kconfig'
+rsource '${ZEPHYR_GOLIOTH_FIRMWARE_SDK_MODULE_DIR}/src/Kconfig'
+
+config GOLIOTH_CIPHERSUITES
```

Comment:
I don't think is being used anywhere, let's remove it.

Reply:
I think we might need this. When I remove it I get this:
```
warning: the int symbol GOLIOTH_ZEPHYR_THREAD_STACK_SIZE (defined at /home/mike/golioth-compile/golioth-firmware-sdk/modules/lib/golioth-firmware-sdk/port/zephyr/Kconfig:121) has a non-int default GOLIOTH_COAP_THREAD_STACK_SIZE (undefined)
```
And I see that's in src/Kconfig:
```
➜ grep GOLIOTH_COAP_THREAD_STACK_SIZE ../../../src/Kconfig -A4
config GOLIOTH_COAP_THREAD_STACK_SIZE
    int "Golioth CoAP thread stack size"
    default 6144
    help
        Thread stack size of the Golioth CoAP thread, in bytes.
```

Reply:
Hmm, it doesn't seem like removing `GOLIOTH_CIPHERSUITES` should cause that particular error.

Reply:
@sam-golioth I was confused by what you were commenting on. I thought you were asking to remove the `rsource` line.

Sorry that I merged before reading this comment. I can make a new PR to remove this block.

Reply:
No worries! We're an agile team 😄

Reply:
Ah, looking into this I remember why I added it. It's a value used in the twister testing:
https://github.com/golioth/golioth-firmware-sdk/blob/8a0825b3f32259287689a6f71b5ee7ad63ea939e/examples/zephyr/hello/sample.yaml#L40-L48

Do want me to remove the GOLIOTH_CIPHERSUITES along with that test?